pkg/cmd/flag/bytes_per_seconds.go

```diff
@@ -0,0 +1,22 @@
+package flag
```

Comment:
Naming nitpick: should the module be `pkg/cli` instead of `pkg/cmd`?

Reply:
Hah, I originally had it under cli and moved it to cmd. I do agree, cli is better

Reply:
Done

Reply:
In fact I introduced a `throttler.BytesPerSec` type to move all the logic into the `throttler` package